examples/glm_reporter_example/reporter_only.py

```diff
@@ -0,0 +1,55 @@
+import os
```

Comment:
the example does not run on my box. Also, these axamples should espect the current organization of nistats examples.

Reply:
They eventually will. I needed a place to stick them somewhere while I am working on the code, where I could test the imports work but out of the way of CircleCI. Once the code and tests are in place, we will decide on the examples.
The current one is the simplest one for me to make sure things are working.
This is Juan's example, use this: https://filesender.renater.fr/?s=download&token=a61fb56c-7c67-a12a-26ae-a2cef64d3ebc

Reply:
No, but... we should go to the simpler solution, which is hust to generate from one of the existing frist-level analysis examples . This would be so much simpler and cleaner !
B

Reply:
Yeah that's what I think as well, but the high-quality examples part will come last, after the functionality & testing has been completed. If we integrate it into current examples right away then CircleCI will keep breaking as I make changes to the code.